blazor/datagrid/infinite-scrolling.md

```diff
@@ -0,0 +1,218 @@
+---
+layout: post
+title: Infinite Scrolling in Blazor DataGrid Component | Syncfusion
+description: Checkout and learn here all about Infinite Scrolling in Syncfusion Blazor DataGrid component and much more details.
+platform: Blazor
+control: DataGrid
+documentation: ug
+---
+
+# Infinite Scrolling in Blazor DataGrid Component
+
+The Infinite Scrolling feature in the DataGrid efficiently manages large data sets without affecting the grid performance. It operates on a "load-on-demand" concept, fetching the buffered data only when necessary, typically in the default infinite scrolling mode, a block of data accumulates every time the scrollbar reaches the end of the scroller.
```

Comment:
The Infinite Scrolling feature in the DataGrid is a powerful tool for seamlessly handling extensive data sets without compromising grid performance. It operates on a "load-on-demand" concept, ensuring that data is fetched only when needed. In the default infinite scrolling mode, a new block of data is loaded each time the scrollbar reaches the end of the scroller. This approach significantly enhances the user experience when working with large data collections in the Blazor DataGrid.

In this mode, a block of data accumulates every time the scrollbar reaches the end of the scroller. To clarify, in this context, a "block" represents the PageSize of the Grid. If the PageSize is not explicitly specified, the Grid will automatically calculate it based on the grid viewport height and row height.

use this content, also give hyper link for pagesize in one place remaining we just to highlight the text.

Reply:
Hi @Venkat-Ayothi ,

Changed as per the suggestion.